tests/PySys/mapper_awaits_before_reconnect/pysystest.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<pysystest type="auto">
+
+  <description>
+	<title>Validate mapper doesn't reconnect too often.</title>
+	<purpose><![CDATA[
+]]>
+	</purpose>
+  </description>
+  <classification>
+	<groups inherit="true">
+	  <group></group>
+	</groups>
+	<modes inherit="true">
+	</modes>
+  </classification>
+  <data>
+	<class name="PySysTest" module="run"/>
```

Comment:
Just a minor request. We usually rename the test Class from PySysTest to a reasonable name (e.g. ValidateReducedReconnectSpeed) which should make it easier to find with an IDE. The name of the test class can be overridden here.